### ISSUE TITLE
Unable add info from VCard

### ISSUE THREAD

**Issue by veselyslon** [thundernest/quicktext#270]
I have filled out a business card for 3 accounts and I can't enter data from it (full name). It works on several computers, some only on some accounts and somewhere at all. Where does Quicktext get this information from?

**Comment by jobisoft** [comment]
For what variable?

**Comment by SamuelPlentz** [comment]
Further Questions:
Is a VCard a normal entry from the address book, or something else? If it is something else what are the steps in Thunderbird to create a VCard?

Are you using any address related addons?

**Comment by veselyslon** [comment]
I copy installation with mozzbackup to other machines and  fill name and work possition on the standard entry in TB. Its same account on each machine, only i I just filled in the correct name of each user.

**Comment by SamuelPlentz** [comment]
It depends from what variable you are using.

The variable `[[TO=fullname]]` uses the fields `firstname` and `lastname` from the address book.
The variable `[[TO=displayname]]` uses the field `displayname` from the address book.

If that doesn't help please answer the questions asked and write in detail what your template looks like and how the fields of a contact (not working like intended) are filled.

**Comment by veselyslon** [comment]
3 accounts, each with a first name and surname in the business card. The same template with a signature 
( <b>[[FROM=fullname]]</b><br>
[[FROM=jobtitle]]<br>) 
in 2 accounts specifies the name, in the last entry the name of the company. However, the name of the company is not mentioned anywhere in the business card, it contains only the name, surname and the name of the job position.

**Comment by jobisoft** [comment]
Can you post a screenshot, where you have added this inforation in TB:

> 3 accounts, each with a first name and surname in the business card.

**Comment by veselyslon** [comment]
![4](https://user-images.githubusercontent.com/25927314/143438709-2b9496bb-b66e-47fe-af54-fe5378915ec7.jpg)
![3](https://user-images.githubusercontent.com/25927314/143438713-45c6b37a-17ba-4b8a-8d3b-14c58c43231d.jpg)
![2](https://user-images.githubusercontent.com/25927314/143438714-275c4b76-001c-4040-acc2-af89aba93ece.jpg)
![1](https://user-images.githubusercontent.com/25927314/143438715-6937209a-b146-470f-b0ed-88cd8cdf7741.jpg)

**Comment by jobisoft** [comment]
I deleted the post as the images contain personal information. I copied the images. I will try to understand. (Not my native language)

**Comment by SamuelPlentz** [comment]
IMO the vCard informations (Thunderbird account settings > edit vCard) are not used at all by Quicktext.
Can you confirm that creating an entry in the address book for the sending email address (with the according fields filled in correctly) solves the problem?

**Comment by veselyslon** [comment]
YES!!!! If exists in contacts entry with my name, Quicktext use this entry, if no, Quicktext use vCard informations (Thunderbird account settings > edit vCard) Problem solved

**Comment by SamuelPlentz** [comment]
IMO we should not include support for that one VCard, but instead rely on the address book. Close?

**Comment by jobisoft** [comment]
The internal storage of contact details changed in 102, Thunderbird now does use the vCard, it is stored and exposed in the _vcard property. The old properties have been removed. Let me think about this.
